doc/src/modules/integrals/integrals.rst

```diff
@@ -46,12 +52,31 @@ Exponential-polynomial functions. These multiplicative combinations of polynomia
 
 even a few nonelementary integrals (in particular, some integrals involving the error function) can be evaluated::
 
-	>>> integrate(exp(-x**2)*erf(x), x)
-	  ____    2
-	\/ pi *erf (x)
-	--------------
-	      4
+	 >>> integrate(exp(-x**2)*erf(x), x)
+   sqrt(pi)*erf(x)**2/4
+
+SymPy's numeric integration (from mpmath) is very accurate and useful and would be preferred
+by many users over using lambdify/scipy. Here is how it is used:
+    >>> from sympy import *
+    >>> x=Symbol('x')
+    >>> integral = Integral(cos(x),(x,0,1))
+    >>> integral
+    Integral(cos(x), (x, 0, 1))
+    >>> integral.evalf()
+    0.841470984807897 
+   
+
+In many cases where symbolic integration is not possible it is still possible to 
+compute the integral numerically. It is also useful as a way of checking the result
+of an integration routine. Integration can be over infinite intervals 
+and can include infinite integrands:
+    >>> from sympy import *
+    >>> x=Symbol('x')
+    >>> Integral(1 / sqrt(x), (x, 0, 1)).evalf()
```

Comment:
Don't put spaces around `/`. In general, you should format expressions the same way `str()` would.